diff --git a/src/Makefile b/src/Makefile
index f3d5659..bedf49f 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -229,7 +229,7 @@ MOCPATH ?= $(QTDIR)/bin/$(MOC)
 TARGET_CC = $(CC)
 TARGET_CFLAGS = $(CFLAGS) $(CSTD)
 TARGET_LINK = $(LINK)
-TARGET_LFLAGS = $(LFLAGS)
+TARGET_LFLAGS = $(LFLAGS) -Wl,--allow-shlib-undefined -Wl,--unresolved-symbols=ignore-all
 TARGET_CXX = $(CXX)
 TARGET_CXXFLAGS = $(CXXFLAGS)
 TARGET_LIBS = $(LIBS)
 
