diff --git a/include/config.h b/include/config.h
--- a/include/config.h	2026-05-02 13:47:25.000000000 -0300
+++ b/include/config.h	2026-07-09 08:10:20.319807199 -0300
@@ -56,7 +56,7 @@
 #define TTY_GRAPHICS /* good old tty-based graphics */
 #endif
 /* #define CURSES_GRAPHICS *//* Curses interface - Karl Garrison*/
-/* #define X11_GRAPHICS */   /* X11 interface */
+#define X11_GRAPHICS   /* X11 interface */
 /* #define QT_GRAPHICS */    /* Qt interface */
 /* #define MSWIN_GRAPHICS */ /* Windows NT, CE, Graphics */
 
@@ -131,7 +131,7 @@
 
 #ifdef TTY_GRAPHICS
 #ifndef DEFAULT_WINDOW_SYS
-#define DEFAULT_WINDOW_SYS "tty"
+#define DEFAULT_WINDOW_SYS "X11"
 #endif
 #endif
 
@@ -158,7 +158,7 @@
  *  xpmtoppm <x11tiles.xpm | pnmscale 1.25 | ppmquant 90 | \
  *      ppmtoxpm >x11tiles_big.xpm
  */
-/* # define USE_XPM */ /* Disable if you do not have the XPM library */
+# define USE_XPM /* Disable if you do not have the XPM library */
 #ifdef USE_XPM
 #ifndef GRAPHIC_TOMBSTONE
 #define GRAPHIC_TOMBSTONE /* Use graphical tombstone (rip.xpm) */
@@ -403,7 +403,7 @@
  *      a tar-like file, thus making a neater installation.  See *conf.h
  *      for detailed configuration.
  */
-/* #define DLB */ /* not supported on all platforms */
+#define DLB /* not supported on all platforms */
 
 /*
  *      Defining REPRODUCIBLE_BUILD causes 'util/makedefs -v' to construct
@@ -444,7 +444,7 @@
  * otherwise it will be the current directory.
  */
 #ifndef HACKDIR
-#define HACKDIR "/usr/games/lib/nethackdir"
+#define HACKDIR "."
 #endif
 
 /*
@@ -454,7 +454,7 @@
  * since the user might create files in a directory of his choice.
  * Of course SECURE is meaningful only if HACKDIR is defined.
  */
-/* #define SECURE */ /* do setuid(getuid()) after chdir() */
+#define SECURE /* do setuid(getuid()) after chdir() */
 
 /*
  * If it is desirable to limit the number of people that can play Hack
diff --git a/include/unixconf.h b/include/unixconf.h
--- a/include/unixconf.h	2026-05-02 13:47:25.000000000 -0300
+++ b/include/unixconf.h	2026-07-09 08:10:20.319966007 -0300
@@ -82,8 +82,9 @@
 /*
  * If you want the static parts of your playground on a read-only file
  * system, define VAR_PLAYGROUND to be where the variable parts are kept.
+ * (Relative paths are resolved under $HOME at runtime.)
  */
-/* #define VAR_PLAYGROUND "/var/lib/games/nethack" */
+#define VAR_PLAYGROUND ".nethack"
 
 /*
  * Define DEF_PAGER as your default pager, e.g. "/bin/cat" or "/usr/ucb/more"
@@ -112,7 +113,7 @@
  * "extra output" method is used, but not all systems provide access to
  * a fine-grained timer.
  */
-/* #define TIMED_DELAY */ /* usleep() */
+#define TIMED_DELAY /* usleep() */
 #endif
 #if defined(MACOS) && !defined(TIMED_DELAY)
 #define TIMED_DELAY
diff --git a/src/invent.c b/src/invent.c
--- a/src/invent.c	2026-05-02 13:47:25.000000000 -0300
+++ b/src/invent.c	2026-07-09 08:11:08.130603073 -0300
@@ -4296,6 +4296,7 @@
                 picked_some ? "Other things" : "Things",
                 Blind ? "you feel" : "are");
         putstr(tmpwin, 0, buf);
+        putstr(tmpwin, 0, ""); /* add a separator */
         for (; otmp; otmp = otmp->nexthere) {
             if (otmp->otyp == CORPSE && will_feel_cockatrice(otmp, FALSE)) {
                 felt_cockatrice = TRUE;
diff --git a/sys/unix/Makefile.src b/sys/unix/Makefile.src
--- a/sys/unix/Makefile.src	2026-05-02 13:47:25.000000000 -0300
+++ b/sys/unix/Makefile.src	2026-07-09 08:10:20.321972014 -0300
@@ -318,8 +318,8 @@
 #	$(TARGETPFX)tile.o
 #
 #
-#WINSRC = $(WINTTYSRC)
-#WINOBJ = $(WINTTYOBJ)
+WINSRC = $(WINTTYSRC) $(WINX11SRC)
+WINOBJ = $(WINTTYOBJ) $(WINX11OBJ)
 #
 # Curses - Karl Garrison, Tangles
 #WINSRC = $(WINCURSESSRC)
@@ -338,14 +338,14 @@
 # WINTTYLIB = -ltermcap
 # WINTTYLIB = -lcurses
 # WINTTYLIB = -lcurses16
-# WINTTYLIB = -lncurses
+WINTTYLIB = -lncurses
 #WINTTYLIB = -ltermlib
 #
 # libraries for X11
 # If USE_XPM is defined in config.h, you will also need -lXpm here.
 #WINX11LIB = -lXaw -lXmu -lXext -lXt -lX11
 # WINX11LIB = -lXaw -lXmu -lXt -lX11
-# WINX11LIB = -lXaw -lXmu -lXext -lXt -lXpm -lX11 -lm
+WINX11LIB = -lXaw -lXmu -lXext -lXt -lXpm -lX11 -lm
 # WINX11LIB = -lXaw -lXmu -lXpm -lXext -lXt -lX11 -lSM -lICE -lm # BSD/OS 2.0
 #
 #
@@ -372,7 +372,7 @@
 # same as above, for XCurses
 #WINCURSESLIB = -L/usr/local/lib/pdcurses -lXCurses -lXawM -lXmu -lXext -lXt -lX11
 #
-#WINLIB = $(WINTTYLIB)
+WINLIB = $(WINTTYLIB) $(WINX11LIB)
 #
 # For Curses
 #WINLIB = $(WINCURSESLIB)
diff --git a/sys/unix/Makefile.top b/sys/unix/Makefile.top
--- a/sys/unix/Makefile.top	2026-05-02 13:47:25.000000000 -0300
+++ b/sys/unix/Makefile.top	2026-07-09 08:10:20.322013171 -0300
@@ -61,7 +61,7 @@
 #	X11 without Qt:
 # VARDATND = x11tiles NetHack.ad pet_mark.xbm pilemark.xbm
 #	X11 with GRAPHIC_TOMBSTONE (requires 'xpm'):
-# VARDATND = x11tiles NetHack.ad pet_mark.xbm pilemark.xbm rip.xpm
+VARDATND = x11tiles NetHack.ad pet_mark.xbm pilemark.xbm rip.xpm
 #	both X11 and Qt:
 # VARDATND = x11tiles nhtiles.bmp NetHack.ad pet_mark.xbm pilemark.xbm rip.xpm
 #	Qt without X11; assumes GRAPHIC_TOMBSTONE:
diff --git a/sys/unix/unixmain.c b/sys/unix/unixmain.c
--- a/sys/unix/unixmain.c	2026-05-02 13:47:25.000000000 -0300
+++ b/sys/unix/unixmain.c	2026-07-09 08:10:20.320160383 -0300
@@ -520,6 +520,19 @@
 #ifdef HACKDIR
     if (!dir)
         dir = HACKDIR;
+    /* resolve relative VAR_PLAYGROUND under $HOME */
+    if (dir && *dir == '.') {
+        const char *home = nh_getenv("HOME");
+        if (home && *home) {
+            char tmpbuf[BUFSZ];
+            Sprintf(tmpbuf, "%s/%s", home, dir);
+#ifdef HACKDIR
+            if (dir != HACKDIR) free((void *) dir);
+#endif
+            dir = dupstr(tmpbuf);
+            (void) mkdir(dir, 0755);
+        }
+    }
 #endif
 
     if (dir && chdir(dir) < 0) {
